addons/metallb/resources/20-speaker-daemonset.yml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1beta2
+kind: DaemonSet
+metadata:
+  namespace: metallb-system
+  name: speaker
```

Comment:
The ability to specify `tolerations` would be much appreciated.

Reply:
@Timer So you'd want only specific nodes to be able to publish the routes? I _think_  you can do the same by targeting the peer configs using proper `node_selectors`.